### PR TITLE
Fishing map e2e/refactor timeouts

### PR DESCRIPTION
**Pull request by fertandil87**
Fixed tests and added custom timeouts

### CODE REVIEW

File: apps/fishing-map-e2e/src/support/app.po.ts [hunk 1]
```diff
@@ -36,6 +36,7 @@ export const disablePopups = () => {
   localStorage.setItem('DisableWelcomePopup', 'true')
   localStorage.setItem('DisableSourceSwitchPopup', 'true')
   localStorage.setItem('HighlightPopup', '"vms-with-png"')
+  localStorage.setItem('WelcomePopup', '{"visible":false,"showAgain":false}')
```
Comment (j8seangel) [inline apps/fishing-map-e2e/src/support/app.po.ts:39]: We have a new popup for the vessel profile and changed the behaviour of the marine manager to follow the same logic.

**Review by j8seangel — APPROVED** [review]
Apart from the comment of the welcome popups, feel free to merge once fixed!

File: apps/fishing-map-e2e/src/support/map/eez-download.po.ts [hunk 2]
```diff
@@ -71,7 +77,8 @@ export const testCsvOptions = (
 ) => {
   cy.log(description)
   deleteDownloadsFolder()
-  buttons.forEach((button) => cy.getBySel(container).findBySel(button).click())
+  // We need force because the buttons are covered by the active element
+  buttons.forEach((button) => cy.getBySel(container).findBySel(button).click({ force: true }))
```
Comment (rdgfuentes) [inline apps/fishing-map-e2e/src/support/map/eez-download.po.ts:81]: If this is used in other places I'd extract it to a custom command as suggested [here](https://stackoverflow.com/a/61325254)
```node
Cypress.Commands.add('forceClick', {prevSubject: 'element'}, (subject, options) => {
  cy.wrap(subject).click({force: true})
});
```
What do you think?

File: apps/fishing-map-e2e/src/e2e/sidebar/search-vessel.cy.ts [hunk 3]
```diff
@@ -36,22 +41,27 @@ xdescribe('Basic search for a vessel', () => {
...
         .then((vessel) => {
           // Click on the first result
-          cy.getBySel('search-vessels-list').findBySelLike('search-vessels-option').eq(0).click()
+          cy.getBySel('search-vessels-list')
+            .findBySelLike('search-vessels-option-selection-0')
+            .click()
           cy.getBySel('search-vessels-add-vessel').click()
           scrollSidebar('center', 2000)
+          console.log(vessel)
+          console.log(vessel.text())
```
Comment (rdgfuentes) [inline apps/fishing-map-e2e/src/e2e/sidebar/search-vessel.cy.ts:62]: Remove these console.logs if not needed 🧹  or use `cy.log` instead.

File: apps/fishing-map-e2e/src/support/app.po.ts (hunk 1 above)
Comment (fertandil87) [reply apps/fishing-map-e2e/src/support/app.po.ts:39]: ok good to know, but it is not necessary right know, I will add it when a test in those pages appear

**Pull request merged**